noaa/track.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+
+async def sql(es, params):
+    logger = logging.getLogger(__name__)
+    logger.info("Provided params [%s].", params)
+    
+    body = params.get("body")
+    # default page_timeout is not suited for benchmarks as it keeps the scroll contexts open for 45s
+    if 'page_timeout' not in body:
+        body['page_timeout'] = '100ms'
+
+    await es.sql.query(
```

Comment:
Depending on the number of results returned, just calling the API via the client can cause significant enough overhead to skew the results. The overhead is due to parsing the entire response on the client side and converting it to JSON. For that reason we have specific code in Rally to avoid touching the response body in any way. I suggest that we move forward with this implementation but I'd like to ask you to raise an issue for Rally to add SQL as a dedicated runner in Rally which does not parse the response body and then use it in this track in a follow-up PR.

Reply:
That's good to know! The same issue applies for `eql`. So I'll raise it for both operations.

Reply:
Thanks for raising https://github.com/elastic/rally/issues/1382.